package.json

```diff
@@ -9,12 +9,17 @@
     },
     "license": "BSD-3-Clause",
     "dependencies": {
+        "bootstrap": "3.3.7",
         "electron": "3.0.2",
         "electron-edge-js": "8.3.8",
         "electron-localshortcut": "3.1.0",
+        "gpii-handlebars": "1.1.4",
         "infusion": "3.0.0-dev.20190328T144119Z.ec44dbfab",
+        "markdown-it": "^8.4.2",
         "nan": "2.10.0",
         "node-jqunit": "1.1.8",
+        "ref": "1.3.4",
```

Comment:
These two dependencies seem unnecessary, they are included by windows and used there, but there shouldn't there be a need to directly reference them from gpii-app

Reply:
Removed!